.git-blame-ignore-revs

```diff
@@ -1 +1,2 @@
 b36196d4ff17934ed71b0926effa20a5a6c1433d
+fff776af314490ed861daedc33d30ae9aeb65532
```

Comment:
Forgot to update the commit SHA.

Now I understand that after rebasing, it doesn’t make much sense to keep this commit here. I’ll revert it and push an updated version soon.

Reply:
Or in a follow up PR?